### ISSUE TITLE
FYI: Is this you?

### ISSUE THREAD

**Issue by daffydock** [TBog/TBLauncher#167]
Just asking, since I see your launcher on the Playstore and instead of linking here, it links here:

https://tbog.github.io/TBLauncher/

Don't know if that is you since I would assume that it would point here, instead. So, just as FYI.

**Comment by ltGuillaume** [comment]
Of course it is. This is the default GitHub page URL.

**Comment by TBog** [comment]
Yes, it's me. It's a link to the GitHub page, not the code page. The first link on that page should be https://github.com/TBog/TBLauncher

Thanks for the information.

**Comment by daffydock** [comment]
It is the link at the Playstore. 

Also, your user/Dev ID is different at the Playstore.  Hence me asking. Over the years, I am sure you know that Kiss Launcher has been copied and illegal store entries have been made.  I do not use the Playstore myself much, hence thought of asking since it would not have surprised me if the same was done for your launcher.

**Comment by TBog** [comment]
Thanks for the heads up.
I tried to divorce my personal Google account from the development account.
